resources/aws/environment.go

```diff
@@ -64,7 +65,8 @@ type Environment struct {
 	awsConfig  *sdkconfig.Config
 	envDefault environmentDefault
 
-	randomSubnets pulumi.StringArrayOutput
+	randomSubnets       pulumi.StringArrayOutput
+	randomFakeintakeIdx pulumi.IntOutput
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
Rather than the random fakeintake, we have a random LB index
```suggestion
	randomLBIdx pulumi.IntOutput
```